src/slider/slider.js

```diff
@@ -30,6 +30,7 @@
     this.element_ = element;
     // Browser feature detection.
     this.isIE_ = window.navigator.msPointerEnabled;
+    this.isEdge_ = !this.isIE_ && !!window.StyleMedia;
```

Comment:
We really shouldn't call this `isEdge` as once edge adds support for `StyleMedia` this isn't going to be accurate anymore.

Is there any way to *feature detect* the actual problem at hand using JS?